_employment/job-seekers/one-on-one.md

```diff
@@ -60,12 +60,6 @@ You might also want to work with nonprofit organizations that have been thorough
 
         <li>
           <h3>
```

Comment:
@mollyblake The first, second, and fourth piano keys now go to page not found. Could you look at the tags again to see how your removal of a piano key affected these others?